include/transaction/signer.h

```diff
@@ -6,7 +6,7 @@
 class Signer
 {
 public:
-    explicit Signer(bytes const &secretKey);
+    explicit Signer(bytes const &seed);
```

Comment:
@AdoAdoAdo, @ccorcoveanu, in erdjs, we call `UserSecretKey` the first 32 bytes of the PEM file, and the decrypted payload of a wallet JSON file (called "seed" in tweetnacl / nacl / libsodium).

https://github.com/ElrondNetwork/elrond-sdk/blob/master/erdjs/src/walletcore/userKeys.ts

Is it all right?

Reply:
In libsodium secret key is called the concatenation of seed and public key, but the public key is only used to save some computation, so I guess it would be alright either way, since it <contains> the same secret.